src/node_dependencies/nested.js

```diff
@@ -60,4 +65,17 @@ const isOptionalModule = function (
   )
 }
 
+// 'critters' is used only in Next.js >= 10.0.4 when enabling an experimental option and has to be installed manually
+// we ignore it if it's missing
+const isExternalModule = function (moduleName, { dependencies = {}, devDependencies = {} }) {
```

Comment:
Should we abstract this by taking the module names and versions from some constant variable such as:

```js
const EXTERNAL_MODULES = {
  critters: { next: '>= 10.0.4' }
}
```

Alternatively we could also wait for another case of external module to abstract, but then maybe we should rename `isExternalModule()` to `isExternalCrittersModule()`?

Reply:
> Alternatively we could also wait for another case of external module to abstract

Let's wait for another use case.
Not sure we want this to be a long term fix anyway.